tests/containers/podman_pods.pm

```diff
@@ -78,9 +78,7 @@ sub run {
         validate_script_output('podman pod ps', sub { !m/testing-pod/ });
         validate_script_output('podman ps', sub { !m/testing-pod-container/ });
 
-        # kube apply
-        # Temporary disabled on TW due to broken tests (See poo#128069 and poo#124601)
-        if (is_sle_micro("5.3+")) {
+        unless (is_sle_micro('<5.4') || is_sle('<15-SP4') || is_leap('<15.4')) {
```

Comment:
Why? Why can't we run this on 15-SP3 for example?

Reply:
The question is: Do we want to run this on 15-SP3?

Reply:
yes, wherever podman is installed and supports this feature, so instead of condition by OS version, I would do condition by podman version.

Reply:
so, maybe using the method `check_min_runtime_version('4.2.0')` ?
@ilausuch @danishprakash do you know since which podman version we have `podman kube apply` command ?

Reply:
Seems to have appeared first in v1.0.0[1][2] as just play and then later matured into `podman {play, generate} kube` in v2.0.0 [3]. I think we're safe to go ahead with more recent versions depending on _what_ we're testing.

[1] - https://github.com/containers/podman/pull/1967
[2] - https://github.com/containers/podman/releases/tag/v1.0.0
[3] - https://github.com/containers/podman/pull/6092/files

Reply:
@danishprakash I don't think so. See [Podman 4.4.0](https://github.com/containers/podman/releases/tag/v4.4.0) release notes:
> A new command, podman kube apply, has been added, which deploys the generated yaml to a k8s cluster.

That in my opinion means that we should do: `check_min_runtime_version('4.4.0')` instead.

Reply:
Oh, I'm sorry about that, I misread that as play for some reason. You're right.

Reply:
No problem :) Fixed now.